decidim-core/app/views/decidim/errors/internal_server_error.html.erb

```diff
@@ -1,10 +1,51 @@
 <div class="wrapper">
   <div class="row">
     <div class="large-8 large-centered">
-    <div class="static__message">
-      <h1 class="heading1 page-title"><%= t(".title") %></h1>
-      <p><%= t(".try_later") %></p>
+      <div class="static__message">
+        <h1 class="heading1 page-title"><%= t(".title") %></h1>
+        <p><%= t(".try_later") %></p>
+        <div class="table-scroll">
+          <table class="table-list">
+            <tr>
+              <th><%= t(".user_id") %></th>
+              <td> <%= user = try(:current_user)&.id || t(".unknown") %></td>
+            </tr>
+            <tr>
+              <th><%= t(".date_and_time") %></th>
+              <td><%= date_and_time = l(Time.current, format: "%Y-%m-%dT%H:%M:%S.%6N") %></td>
+            </tr>
+            <tr>
+              <th><%= t(".url") %></th>
+              <td><%= request_url = try(:request).original_url %></td>
+            </tr>
+            <tr>
+              <th><%= t(".request_method") %></th>
+              <td><%= @method %></td>
+            </tr>
+            <tr>
+              <th><%= t(".reference") %></th>
+              <td><%= reference_id = Decidim::LogReferenceGenerator.new(request).generate_reference %></td>
```

Comment:
Could you move the reference generation to the controller and pass it to the view using an instance variable, the same way as you are doing e.g. with the `@method` variable right now?